src/ZkFold/Symbolic/Compiler/ArithmeticCircuit/Instance.hs

```diff
@@ -203,3 +219,25 @@ instance (FromJSON a, KnownNat n) => FromJSON (ArithmeticCircuit n a) where
                 acOutput  = Vector outs
                 acCircuit = Circuit{..}
             pure ArithmeticCircuit{..}
+
+data ArithmeticCircuitTest n a = ArithmeticCircuitTest
```

Comment:
Does this type have any usecases except for testing? Maybe we can move it closer to its actual place of usage in tests along with `mapVar...` function?
@vlasin, does `mapVarArithmeticCircuit` have any intended usages outside of testing?

Reply:
Yes, agree.

> @vlasin, does `mapVarArithmeticCircuit` have any intended usages outside of testing?

Nope. Previously, it did, but it wasn't strictly necessary.